wlp-managed-8.5/src/main/java/org/jboss/arquillian/container/was/wlp_managed_8_5/WLPManagedContainer.java

```diff
@@ -856,6 +860,37 @@ public void undeploy(Descriptor descriptor) throws DeploymentException {
 
    }
 
+   private void checkForDefinitionExceptions(String applicationName)
+   {
+      String messagesFile = containerConfiguration.getWlpHome() + "/usr/servers/" + containerConfiguration.getServerName() + "/logs/messages.log";
+      BufferedReader br = null;
+
+      try {
+         br = new BufferedReader(new InputStreamReader(new FileInputStream(messagesFile)));
+         String line;
+         while ((line = br.readLine()) != null) {
+            if (line.contains("CWWKZ0002E: An exception occurred while starting the application " + applicationName + ".")
```

Comment:
I assume the text for this message is internationalized; do you think we an restrict this only to the error code without the message or find out what the message should be for the currently active locale?

Reply:
Good point.